lib/grpnames.gd

```diff
@@ -16,6 +16,26 @@
 
 #############################################################################
 ##
+#O  IsTrivialNormalIntersection( <G>, <U>, <V> ) . . . . . . . generic method
+##
+##  <ManSection>
+##  <Oper Name="IsTrivialNormalIntersection" Arg="G, U, V"/>
+##
+##  <Description>
+##    For normal subgroups U and V of G, IsTrivialNormalIntersection returns
+##    true if U and V intersect trivially, and false otherwise.
+##    If U and V are not normal subgroups of G, then it may return true
+##    even if they have a nontrivial intersection.
+##  </Description>
+##  </ManSection>
+##
+DeclareOperation( "IsTrivialNormalIntersection",[IsGroup, IsGroup, IsGroup]);
+# checks if arg2 and arg3 both contain the first generator of any element in
+# the given list arg1 of (minimal) normal subgroups
+DeclareOperation( "IsTrivialNormalIntersectionInList",[IsList, IsGroup, IsGroup]);
+
```

Comment:
I do not understand the documentation comment. Is this a function that is supposed to be used by end users? If so, it should be properly documented. If not, I'd say it shouldn't be declared as an operation.
Also, the formatting is off here, too.